topic5/movies/src/components/AddComponent.jsx

```diff
@@ -0,0 +1,90 @@
+import React,{Component} from 'react';
+import Movie from './Movie.jsx';
+
+
+class AddComponent extends Component {
+	constructor(props){
+		super(props);
+		this.state = {
+            movieText: '',
+            movies: [],
+		};
+	}
+
+updateMovieText(movieText){
+    this.setState({movieText: movieText.target.value})
+}
+
+addMovie(){
+    if(this.state.movieText === ''){return}
```

Comment:
You should avoid doing this: a function should always return something. 

You can change this by doing if(!this.state.movieText) {  here you put what the function does }